### ISSUE TITLE
[Tracking Issue] Relax high-level operator migration

### ISSUE THREAD

**Issue by MasterJH5574** [mlc-ai/relax#62]
Operator migration:
- [x] Unary / binary / ternary tensor arithmetic and comparison operators #61
- [x] Neural network operators
  - [x] NN1. Conv2D, MaxPool2D, AdaptiveAvgPool2D, ReLU, GeLU, SiLU, Softmax #67
  - [x] NN2. BatchNorm, LayerNorm, Matmul, Dropout, CrossEntropy #72
- [x] Tensor statistical operators #73
- [x] Image operators #73
- [x] Tensor manipulation operators
  - [x] Manip1. Reshape, Transpose, ExpandDims, Squeeze, Flatten #78
  - [x] Manip2. Concat, Split #80 
  - [x] Manip3. BroadcastTo #86
- [x] Tensor creation operators
  - [x] Create1. Full/FullLike, Zeros/ZerosLike, Ones/OnesLike #79
  - [x] Create2. Tril, Triu #86
- [x] Tensor indexing operators
  - [x] Take, StridedSlice #85
- [x] Set operators
  - [x] Unique tlc-pack/relax#339
- [x] Searching operators
  - [x] Where #86

Frontend and transformation:
- [x] PyTorch translator #95 
- [x] Operator legalizer #96

The branch with latest work progress is https://github.com/MasterJH5574/relax/commits/mlc-dev/2022-12-24-op-with-struct-info-backup
